### PR TITLE
Voting

### PR DESCRIPTION
**Pull request by jc-roll**

### CODE REVIEW

File: src/models/voter.js [hunk 1]
```diff
@@ -1,139 +1,68 @@
...
- * @description Accepts the parameters and then updates table data
- * @param payload 
- * Holds the req.body in a variable
- * @param err
- * Errors
- * @success
- * Run  
-*/
+  function _update(payload, err, success) { // Accepts the parameters and then updates table data
     var cleanData = util.scrubData(payload);;
```
Comment (ptlergo) [inline src/models/voter.js]: double semicolon #46

File: src/models/voter.js [hunk 2]
```diff
@@ -1,139 +1,68 @@
...
- * @description Cleans and creates the data for voting in DB using the payload
- * @param payload 
- * Holds the req.body in a variable
- * @param err
- * Errors
- * @success
- * Run  
-*/
+  function _create(payload, err, success) { // Accepts the parameters and then creates the data for voting in DB using the payload
     var cleanData = util.scrubData(payload);;
```
Comment (ptlergo) [inline src/models/voter.js]: double semicolon #46

Comment (jc-roll) [inline src/models/voter.js]: Thank you

File: src/models/voter.js (hunk 1 above)
Comment (jc-roll) [inline src/models/voter.js]: Thank you

File: package.json [hunk 3]
```diff
@@ -18,6 +18,7 @@
     "jsdoc": "3.4.0",
     "moment": "^2.13.0",
     "mysql": "2.10.2",
+    "path": "^0.12.7",
```
Comment (reactivepixel) [inline package.json]: No ^s

File: src/models/history.js [hunk 4]
```diff
@@ -1,47 +1,44 @@
...
-
-
+            /**
+              * @function _create
+              * @description Cleans the data within voting in DB
+              * @param payload 
+              * Holds the req.body in a variable
+              * @param err
+              * Errors
+              * @success
```
Comment (reactivepixel) [inline src/models/history.js]: @success ?

File: src/models/history.js [hunk 5]
```diff
@@ -1,47 +1,44 @@
...
   }
-
+            /**
+              * @function _update
+              * @description Accepts the parameters and then updates table data
+              * @param payload 
+              * Holds the req.body in a variable
+              * @param err
+              * Errors
+              * @success
```
Comment (reactivepixel) [inline src/models/history.js]: @param

File: src/models/voter.js [hunk 6]
```diff
@@ -1,118 +1,69 @@
...
+  function _update(payload, err, success) { // Accepts the parameters and then updates table data
+    var cleanData = util.scrubData(payload);
     db.voting.find({
       where: {
-        id: cleanData.id  /** varifies that the ID's match */
+        id: cleanData.id       
       }
     }).then(function(matchedvoting) {
-      matchedvoting.updateAttributes(cleanData).then(success).catch(err); /** Updates information using updateAttributes */
+      matchedvoting.updateAttributes(cleanData).then(success).catch(err); // Updates information using updateAttributes
```
Comment (reactivepixel) [inline src/models/voter.js:31]: CamelCase

File: src/routes/api/voting.js [hunk 7]
```diff
@@ -1,89 +1,86 @@
...
-    req.body.id = req.params.vote_id;
+    /**
+      * @property req.body
+      * @description holds parameters that are sent up from the client as part of a POST request 
+    */
+    req.body.id = 
+                  /**
+                    * @property req.params
+                    * @description will return parameters in the matched route
+                  */
```
Comment (reactivepixel) [inline src/routes/api/voting.js]: Don't let comments interfere with setting properties.

File: src/server.js [hunk 8]
```diff
@@ -3,7 +3,7 @@
  * This is the express module being put into a variable, express.
 */
 var express = require('express'); // Sets variable express to express module
-var util = require('../lib/util');
+// var util = require('../lib/util');
```
Comment (reactivepixel) [inline src/server.js]: Remove, don't just comment.

**Pull request merged**